packages/cli/src/commands/start/command.ts

```diff
@@ -31,10 +31,14 @@ export async function command(opts: OptionValues): Promise<void> {
 
   switch (role) {
     case 'backend':
+      return startBackend(options);
+    case 'node-library':
     case 'backend-plugin':
     case 'backend-plugin-module':
-    case 'node-library':
-      return startBackend(options);
+      return startBackend({
+        ...options,
+        entry: 'dev/index',
```

Comment:
This already exists in the form of [`src/run.ts`](https://github.com/backstage/backstage/blob/0efd2f4b6159dbd519338247160910e5d07e42e2/packages/cli/src/lib/bundler/paths.ts#L55). That's very old and I do like this more, but it's a breaking change. I think it's best to move over the dev setup to that file for now, and then if you want to we can discuss switching that over separately?

Reply:
Ah right. Yeah thanks, will revert